consoleme/config/config.py

```diff
@@ -123,9 +123,23 @@ async def merge_extended_paths(self, extends, dir_path):
             if extend_config.get("extends"):
                 await self.merge_extended_paths(extend_config.get("extends"), dir_path)
 
-    async def load_config(self):
+    def reload_config(self):
+        # We don't want to start additional background threads when we're reloading static configuration.
+        while True:
+            async_to_sync(self.load_config)(
+                allow_automatically_reload_configuration=False,
+                allow_start_background_threads=False,
+            )
+            time.sleep(self.get("dynamic_config.reload_static_config_interval", 60))
```

Comment:
Should we break out of this loop if the value of `"config.automatically_reload_configuration"` changes instead of `while True`? Like if someone starts the instance with it set to True to test things out and it's ready, so they want to turn off automatic reloading of static config?

Reply:
Yes, nice find